profile_helper.fish

```diff
@@ -19,7 +19,9 @@ set BASE16_SHELL_THEME_NAME_PATH "$BASE16_CONFIG_PATH/theme_name"
 # Allow users to optionally configure their base16-shell path and set
 # the value if one doesn't exist
 if test -z $BASE16_SHELL_PATH
+  set -l CWD (pwd)
   set -g BASE16_SHELL_PATH (cd (dirname (status -f)); and pwd)
+  cd "$CWD"
 end
```

Comment:
This can probably be `set -g BASE16_SHELL_PATH (realpath (dirname (status -f))`. `realpath` will make the path absolute *and* resolve symlinks in between which I imagine is what the `cd` is trying to account for.

Reply:
Sure, that's sensible. Pushed.